addons/field_rations/stringtable.xml

```diff
@@ -485,7 +511,7 @@
             <Chinesesimp>食堂</Chinesesimp>
             <Czech>Jídelna</Czech>
             <French>Cantine</French>
-            <German>Kantine</German>
+            <German>Feldflasche</German>
```

Comment:
???

Reply:
Habe mitbekommen wie viele Leute das Wort Kantine hörten und nicht ganz verstanden was damit gemeint war, sowie hört sich Kantine etwas zu direkt übersetzt an und Feldflasche ist ebenso der direkt genutzte Begriff im Militär für ein Trinkgefäß was man im Feld mitführt. (Was es ja hier darstellen soll)
Er ist also dadurch verständlicher weil eindeutig und in dem Raum was es darstellen will bekannter als nur Kantine durch seiner omnipräsenten Begriffsnutzung in der BW.